Main.java

```diff
@@ -0,0 +1,20 @@
+package com.company;
+
+public class Main {
+
+    public static void main(String[] args) {
+        InvestmentCompany Berkshire_Partners = new InvestmentCompany();
```

Comment:
Berkshire_Partners variable name does not match code convention requirements

Reply:
fixed